scripts/cypress.js

```diff
@@ -8,20 +8,34 @@ const path = require('path');
  * https://github.com/cypress-io/cypress/issues/5674
  */
 
+const argv = require('yargs')
+  .option('mode', {
+    describe: 'Choose a mode to run cypress',
+    choices: ['run', 'open'],
+  })
+  .option('port', {
```

Comment:
why was this needed as part of the fix? I'd be great if we can dissect adding functionality in separate PRs going forward

Reply:
I added it while debugging, to help with cases where storybook had already started on a different port. But I agree doing it separately makes sense in general.